src/biokbase/narrative/jobs/util.py

```diff
@@ -50,37 +50,3 @@ def load_job_constants(relative_path_to_file=JOB_CONFIG_FILE_PATH_PARTS):
             )
 
     return (config["params"], config["message_types"])
-
-
-def sanitize_state(state):
```

Comment:
no mention of these methods anywhere else in the codebase (apart from tests) and looking at the test data, the job data structures have changed significantly since these tests were written and this code was used. 

==> DELETE!